<ui version="4.0" >
 <author>Pere Constans</author>
 <class>c2bConfigureFR</class>
 <widget class="QDialog" name="c2bConfigureFR" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>151</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Edit Find-Replace - cb2Bib</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>6</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="1" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="layoutDirection" >
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>271</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="Line" name="line2" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="2" column="0" >
      <widget class="QLabel" name="textLabel3" >
       <property name="text" >
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="textLabel1" >
       <property name="text" >
        <string>Find:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="textLabel2" >
       <property name="text" >
        <string>Replace:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="c2bLineEdit" name="lineEditFind" />
     </item>
     <item row="1" column="1" >
      <widget class="c2bLineEdit" name="lineEditReplace" />
     </item>
     <item row="2" column="1" >
      <widget class="c2bLineEdit" name="lineEditDescription" />
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>c2bLineEdit</class>
   <extends>QLineEdit</extends>
   <header>c2bLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditFind</tabstop>
  <tabstop>lineEditReplace</tabstop>
  <tabstop>lineEditDescription</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>c2bConfigureFR</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>637</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel" >
     <x>104</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>c2bConfigureFR</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>558</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel" >
     <x>24</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
